index.html

```diff
@@ -10,6 +10,68 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
-  <h2 class="title">Back-End Tools</h2>
+  <div class="boxes">
+    <div class="box">
+      <img class="box-cover" src="assets/react_icon.svg" alt="">
```

Comment:
this should be a self closing tag. use  `/>` at the end to explicitly say this tag has no content. also add an alt text that represents the image content `react logo` would be a good choice.